README.md

```diff
@@ -18,7 +18,7 @@ First, install [node][1] if you don't have it, then install the mParticle comman
 **Note:** Currently, linting requires that you to use version 1.0.3 of the mParticle CLI. If you have a later version, please uninstall the cli and reinstall version 1.0.3.
```

Comment:
Looks like this version needs to be updated as well

Reply:
Do we still need this note, or can we change it to something that doesn't reference the actual version number?

Reply:
Ah yeah actually this can just be removed, I think?